lib/govuk_publishing_components/app_helpers/countdown_helper.rb

```diff
@@ -0,0 +1,33 @@
+module GovukPublishingComponents
+  module AppHelpers
+    class CountdownHelper
+      END_OF_TRANSITION_PERIOD = Date.new(2021, 1, 1)
+
+      def days_left
+        sprintf "%02d", days_left_integer
+      end
+
+      def show?
+        days_left_integer.positive?
+      end
+
+      def days_text
+        if days_left_integer == 1
+          I18n.t("components.transition_countdown.day_to_go")
+        else
+          I18n.t("components.transition_countdown.days_to_go")
+        end
+      end
+
+    private
+
+      def days_left_integer
+        (END_OF_TRANSITION_PERIOD - today_in_london).to_i
+      end
+
+      def today_in_london
+        Time.zone.today
```

Comment:
`Time.zone` isn't guaranteed to be London (in fact we only recently [documented it as a convention](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#specify-london-as-the-timezone) after discovering quite a few apps aren't in London).

I think you probably want `Time.find_zone("Europe/London").today` - although that's just a result of a quick Google, I'm not sure it's the most idiomatic way to define it.

Reply:
Yes, I was aware of the recently documented convention and checked the applications that are currently calling the component to have the right time zone set, but this seems more robust. Have updated the code.

Reply:
Yes this worked in collections as the [app is set to London time](https://github.com/alphagov/collections/pull/1997)